src/App.tsx

```diff
@@ -313,11 +327,7 @@ function App() {
           />
         </div>
         <div className="col-span-3">
-          <ResultContainer
-            result={
-              result ? result.sort((a, b) => a.dataset_name.localeCompare(b.dataset_name)) : null
```

Comment:
Ah, yeah, should probably bring that back! Good call. I removed it because the type changed so sorting gets more complex (has to happen on a specific key inside the array) - but still feasible.